apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingFactory.kt

```diff
@@ -112,20 +117,31 @@ class BehandlingFactory(
                 mottattDato = mottattDato,
                 kilde = kilde,
                 revurderingAarsak = RevurderingAarsak.NY_SOEKNAD,
-            )
+            )?.let { BehandlingOgOppgave(it, null) }
```

Comment:
Ser for meg at regulering vil ha denne oppgaven siden vi snakker om automatisk revurdering her -- men det er kanskje litt utenfor scopet du ser på nå uansett?